examples/out_dir/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@aspect_rules_swc//swc:defs.bzl", "swc")
+
+swc(
+    name = "transpile",
+    srcs = [
+        "a.ts",
+        "b.ts",
+    ],
+    out_dir = "out",
+    source_maps = "true",
+)
+
+# Since the srcs were in a filegroup, the swc macro cannot pre-declare the outputs.
+# So there is no label ":a.js" that we can reference from the build file.
+# However, a.js is still produced as one of the default outputs of the transpile rule.
+# We can verify this in an action that depends on the ":transpile" rule and reads the files.
+sh_test(
```

Comment:
nit: you could just use https://github.com/aspect-build/bazel-lib/blob/main/lib/testing.bzl#L43 if you don't need to assert the contents

Reply:
I guess for this test we don't care about the content. I'll update 👍